### PR TITLE
Update Tencent_CN.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

**Comment by renyouguo** [comment]
@jeremyn I have tried to clear the comments.

File: src/chrome/content/rules/Tencent_CN.xml [hunk 1]
```diff
@@ -1,22 +1,45 @@
 <!--
-	Might be blocked by The Great Fire Wall (GFW) in China, Check error: Connection timed out ( after 20000 milliseconds ).
+
+	- Might be blocked by The Great Fire Wall (GFW) in China, Check error: Connection timed out ( after 20000 milliseconds ).
 	Test in Tor broswer 6.0.4, Ok by http, however, unable to connect to https.
 	Users could turn on this rule to test by youself.
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Tencent_CN.xml]: Typo fix: you(r)self

**Comment by jeremyn** [comment]
@ivysrono I saw your frustration about this issue in https://github.com/EFForg/https-everywhere/issues/7242#issuecomment-250891917 .

This looks really tough to review for reasons like:

* "Might be blocked" by the GFW, I don't know how to test that.
* Some unusual behavior in Tor?
* Marked `default_off="Only avaiable in China"`, do I need to use a VPN that outputs in China? Is Hong Kong okay?
* A long comment talking about special RSS and pic behavior and modifying URLs that maybe requires me to download a Chinese app. I don't know how to test that.
* There are redirects from `iwgc.cn` to `qpic.cn`. I don't like redirects anyway (see https://github.com/EFForg/https-everywhere/issues/7253#issuecomment-250918673 ), but here I don't know what these sites are and I can't read them so I don't know how to verify this is okay.

**Please note** I'm not asking you to answer these questions right now, I'm just showing you some things I see as a reviewer that make this PR look very hard or impossible to review for me. Unfortunately, because we have a lot of open PRs, "difficult" PRs can get overlooked.

For this specific PR, I don't know if anyone with merge access is able to review this correctly, because it looks so specific to the Chinese internet. I know that's not fair to you. We should have a wider range of volunteers, including reviewers, from more countries, for example see the discussion starting at https://github.com/EFForg/https-everywhere/issues/6424#issuecomment-241224427 .

**Comment by gloomy-ghost** [comment]
@jeremyn In this case, `blocked by GFW` means the https version is inaccessible outside China. I think @ivysrono is using Tor to emulate the environment outside China, and Tor can be changed to anything which does the same job.

If you want to confirm that it is accessible in China, just use greatfire analyzer. 
e.g. https://en.greatfire.org/https/dp3.qq.com

For RSS and pic, they are unofficial things. There are some privacy issues, and I'm not sure it is okay or not.

**Comment by jeremyn** [comment]
Thanks @gloomy-ghost , but I want to again emphasize that my last comment was just to explain to @ivysrono why this PR seems difficult to review and so why it might be getting "ignored". I'm not volunteering to review this one myself and I don't want @ivysrono to put a ton more effort into this and get disappointed.

**Comment by ivysrono** [comment]
@jeremyn First of all, thank you very much for your patience.
For # 6889 # 6894, I am currently using Redirector extension, for me this is no issue.
To submit these two PRs, I want to make sure that # 6894 works on the mobile version of Firefox for all users, and secondly that I want to know what your policies or attitudes about these special situations are.
The merger?  Not the most important, I just want to be fully discussed.

**Comment by jeremyn** [comment]
This PR is special in different ways. I'm not sure what you're asking. Can you ask your question in a new issue?

**Pull request closed without merging**